mubench.reviewsite/templates/review.phtml

```diff
@@ -335,6 +397,18 @@ use MuBench\ReviewSite\Model\Experiment;
                     }
                     b.type = "hidden";
                 }
+                function onInput() {
+                    console.log("Hello");
+                    var val = document.getElementById("tag-input").value;
+                    var opts = document.getElementById("tags").childNodes;
+                    var form = document.getElementById("tag-form");
+                    for (var i = 0; i < opts.length; i++) {
```

Comment:
It's not clear to me what this loop does. Can you make it more clear or add an explanatory comment?